hols/oci-basic-setup/README.md

```diff
@@ -0,0 +1,203 @@
+1. Open a Cloud Shell terminal and make sure you are in the home directory
+   ```shell
+   cd ~
+   ```
+2. Use `git sparsecheckout` to check out only the `oci-basic-setup` directory from the `helidon-labs` repository.
+   ```shell
+   git init helidon-labs
+   cd helidon-labs 
+   git remote add origin https://github.com/helidon-io/helidon-labs.git
+   git config core.sparsecheckout true
+   echo "hols/oci-basic-setup/*" >> .git/info/sparse-checkout
+   echo ".gitignore" >> .git/info/sparse-checkout
+   git pull --depth=1 origin main
+   ```
+3. The previous step pulls all the required Terraform and Bash script files into the  `~/helidon-labs/hols/oci-basic-setup` directory that is required to perform the various operations in this exercise.
+
+### Prepare the OCI infrastructure environment
+The goal of this task is to prepare a basic infrastructure environment comprised of a Compartment, Dynamic Groups, Policies, Compute and Virtual Cloud Network. This section requires a user with administrator privilege.
+1. Open the newly cloned repository directory `oci-basic-setup` from the a Cloud Shell terminal.
+   ```shell
+   cd ~/helidon-labs/hols/oci-basic-setup
+   ```
+2. From the root directory of the repository, open `terraform.tfvars` and set the values of the following variables: 
+   * `tenancy_ocid` - This can be retrieved by opening the `Profile` menu (click the User icon on the top most right corner of the console window) and click `Tenancy: <your_tenancy_name>`. The tenancy OCID is shown under `Tenancy Information`. Click `Show` to display the entire ID or click `Copy` to copy it to your clipboard.
+   * `region` - From OCI Console's home screen, open the `Region` menu, and then click `Manage Regions`. Locate the Home Region and copy the Region identifier.
+   * `instance_*` - Use these parameters to specify the size and shape of the OCI Compute instance that you would like to be provisioned. Currently defaults to an available shape from an [Always Free Resources of an OCI Free Tier Tenancy](https://docs.oracle.com/en-us/iaas/Content/FreeTier/freetier_topic-Always_Free_Resources.htm).
+     ```terraform
+     instance_shape                      = "VM.Standard.E2.1.Micro"
+     instance_ocpus                      = 1
+     instance_shape_config_memory_in_gbs = 1
+     instance_os                         = "Oracle Linux"
+     instance_os_version                 = "8"
+     ```
+4. Use Terraform to execute the scripts to provision the OCI resources.
+   ```bash
+   terraform init
+   terraform plan
+   terraform apply -auto-approve
+   ```
+    
+### Prepare the Helidon OCI Application:
+1. Go to the home directory from a Cloud Shell console:
+   ```shell
+   cd ~
+   ```  
+2. Download and unzip the Helidon CLI generic distribution
+   ```shell
+   curl -L -O https://github.com/helidon-io/helidon-build-tools/releases/download/3.0.6/helidon-cli.zip
+   unzip helidon-cli.zip
```

Comment:
Consider adding ~/helidon-3.0.6/bin to PATH (and maybe a suggestion to add this to ~/.bashrc).
Note, this may be a maintenance issue as it will change whenever a new version of the cli is created.

Reply:
Updated